notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        curr_flux = fits.open(file)[1].data["FLUX"].ravel()
</pre><p> </p><p>Let's use a context manager for lines 6 and 7.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>